public/resources/remix/style.less

```diff
@@ -6,7 +6,7 @@
 
 @font-face {
   font-family: "Fira Medium";
-  src: ~"url(FiraSansOT-Regular.otf)";
```

Comment:
@gideonthomas I'm not exactly certain about why I wrote it this way. It might be that "Medium" as a relative description of font-weight within the UI just maps better to the "Regular" version of the font. "Fira Medium" is just a cosmetic label anyway - so I wouldn't worry about this discrepancy.